ocsp.go

```diff
@@ -38,16 +40,24 @@ var cacheDir = ""
 // cacheFileName is the file name of OCSP response cache file
 var cacheFileName = ""
 
+// cacheUpdated is true if the memory cache is updated
+var cacheUpdated = true
+
+// OCSP fail open mode
+var ocspFailOpen = ocspFailOpenTrue
+
 const (
 	// retryOCSPTimeout is the total timeout for OCSP checks.
-	retryOCSPTimeout     = 120 * time.Second
-	retryOCSPHTTPTimeout = 30 * time.Second
+	retryOCSPTimeout     = 60 * time.Second
+	retryOCSPHTTPTimeout = 20 * time.Second
 )
 
 const (
-	cacheFileBaseName = "ocsp_response_cache"
+	cacheFileBaseName = "ocsp_response_cache.json"
```

Comment:
This change allows the Golang driver to share the cache file with other Snowflake drivers.